index.html

```diff
@@ -292,6 +292,15 @@ <h3 class="modal__title" data-translate="__MSG_SETTINGS__">Settings</h3>
                     </div>
                 </div>
             </fieldset>
+            <fieldset class="selector">
+                <legend class="selector__label" data-translate="__MSG_ARCHIVE_ITEM_AFTER_OPEN__">
+                    Archive Item after open?
+                </legend>
+                <div class="selector__item">
+                    <input type="checkbox" name="archive-after-open" id="archive-after-open" class="cursor-pointer">
+                    <label for="archive-after-open" class="cursor-pointer">Enable</label>
```

Comment:
Enable text should come from `messages.json` file too.

Reply:
Done.